templates/viewer/notes.html

```diff
@@ -0,0 +1,50 @@
+<!-- THE NOTE CARDS -->
```

Comment:
Is this file any different than the copy of it in the main template? If not, why have you recopied it here?

Reply:
I change all the variable in viewer.notes.html. I use passage_data because i have only one text when i used viewer.multiviewer.html